builder/index.html

```diff
@@ -394,16 +391,15 @@ <h4 class="panel-title">
                             </li>
                         </ul>
                     </div>
-                    <div class="col-md-3" class="bs-sidebar navbar" >
-                        <ul class="nav bs-sidenav" style="position:fixed; margin-top:68px;">
+                    <div class="col-lg-2 col-md-2 col-sm-2 col-xs-4 ncf_col" class="bs-sidebar navbar" ng-if="selectedMethod === undefined">
```

Comment:
I think you could have col-lg-1 col-md1 (and inscrease on to col-lg-4 on the methods) and reduce the padding on that column.

It's better to have more place on the left side